java-squid/src/main/java/org/sonar/java/se/SymbolicExecutionVisitor.java

```diff
@@ -38,7 +39,10 @@
   @Override
   public void visitNode(Tree tree) {
     try {
-      tree.accept(new ExplodedGraphWalker(context));
+      System.out.println(context.getFile().getName()+"#"+ ((MethodTree) tree).simpleName().name());
+      ExplodedGraphWalker visitor = new ExplodedGraphWalker(context);
+      tree.accept(visitor);
+      System.out.println("--->"+visitor.steps);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS106)